.github/workflows/image-vscode-ssh-server.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Build VSCode SSH Server
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/image-vscode-ssh-server.yml
+      - images/vscode-ssh-server/**
+    tags:
+      - vscode-ssh-server@v*
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/image-vscode-ssh-server.yml
+      - images/vscode-ssh-server/**
+  release:
+    types:
+      - created
```

Comment:
Doing this off of tags is sufficient; I have a PR out to remove this from the kairos image.

Reply:
I don't see the PR but I'll strip this out